File: app/src/main/java/com/gmail/fattazzo/formula1world/news/reader/Formula1ItNewsReader.java
package com.gmail.fattazzo.formula1world.news.reader;

import org.jsoup.nodes.Element;

/**
 * @author fattazzo
 *         <p/>
 *         date: 21/07/17
 */
public class Formula1ItNewsReader extends RssNewsReader {

    @Override
    protected String getChannelImage(Element channelElement) {
        return "assets://news/formula1itlogo.png";
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/news/reader/Formula1ItNewsReader.kt
package com.gmail.fattazzo.formula1world.news.reader

import org.jsoup.nodes.Element

/**
 * @author fattazzo
 *
 *
 * date: 21/07/17
 */
class Formula1ItNewsReader : RssNewsReader() {

    override fun getChannelImage(channelElement: Element): String {
        return "assets://news/formula1itlogo.png"
    }
}
